setup.py

```diff
@@ -164,7 +164,7 @@ def find_package_data(
     zip_safe=False,
     install_requires=[
         "django-appconf >= 1.0.3",
-        "rcssmin == 1.1.1",
+        "rcssmin == 1.1.2",
         "rjsmin == 1.2.1",
```

Comment:
```suggestion
        "rjsmin == 1.2.2",
```
Also new version of `rjsmin`, might as well bump both.